app/actions/websocket/users.ts

```diff
@@ -120,3 +123,129 @@ export async function handleStatusChangedEvent(serverUrl: string, msg: WebSocket
     const newStatus = msg.data.status;
     setCurrentUserStatus(serverUrl, newStatus);
 }
+
+export async function handleCustomProfileAttributesValuesUpdatedEvent(serverUrl: string, msg: WebSocketMessage) {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
+        logError('No operator found');
+        return;
+    }
```

Comment:
you can wrap the whole thing in a try/catch and use DatabaseManager.getServerDatabaseAndOperator instead, same for all functions in this file